sections/leaderboard/TraderHistory.tsx

```diff
@@ -0,0 +1,398 @@
+								),
+								accessor: 'title',
+								columns: [
+									{
+										Header: (
+											<TableHeader>{t('leaderboard.trader-history.table.market')}</TableHeader>
+										),
+										accessor: 'asset',
+										Cell: (cellProps: CellProps<any>) => (
+											<CurrencyInfo>
+												<StyledCurrencyIcon currencyKey={cellProps.row.original.currencyIconKey} />
+												<StyledSubtitle>{cellProps.row.original.marketShortName}</StyledSubtitle>
+												<StyledFuturesIcon type={cellProps.row.original.accountType} />
+											</CurrencyInfo>
+										),
+										width: 50,
+									},
+									{
+										Header: (
+											<TableHeader>{t('leaderboard.trader-history.table.status')}</TableHeader>
+										),
+										accessor: 'status',
+										Cell: (cellProps: CellProps<any>) => {
+											return <StyledCell>{cellProps.row.original.status}</StyledCell>;
+										},
+										width: 30,
+									},
+									{
+										Header: (
+											<TableHeader>{t('leaderboard.trader-history.table.total-pnl')}</TableHeader>
+										),
+										accessor: 'pnl',
+										Cell: (cellProps: CellProps<any>) => (
+											<PnlContainer>
+												<ColorCodedPrice
+													currencyKey={'sUSD'}
+													price={cellProps.row.original.pnl}
+													sign={'$'}
+													conversionRate={1}
+												/>
+												<StyledValue
+													color={
+														cellProps.row.original.pnl.gt(0)
+															? 'green'
+															: cellProps.row.original.pnl.lt(0)
+															? 'red'
+															: ''
+													}
+												>
+													{cellProps.row.original.pnlPct}
+												</StyledValue>
+											</PnlContainer>
+										),
+										width: 40,
+										sortType: 'basic',
+										sortable: true,
+									},
+								],
+							},
+						]}
+					/>
+				</MobileOrTabletView>
+			</>
+		);
+	}
+);
+
+const StyledTable = styled(Table)<{ compact: boolean | undefined }>`
+	margin-top: ${({ compact }) => (compact ? '0' : '15px')};
+`;
+
+const TableTitle = styled.div`
+	width: 100%;
+	display: flex;
+	justify-content: flex-start;
+`;
+
+const TitleText = styled.a`
+	font-family: ${(props) => props.theme.fonts.regular};
+	color: ${(props) => props.theme.colors.selectedTheme.gray};
+
+	&:hover {
+		text-decoration: underline;
+		cursor: pointer;
+	}
+`;
+
+const StyledCell = styled.div`
+	color: ${(props) => props.theme.colors.selectedTheme.button.text.primary};
+	display: flex;
+`;
+
+const TitleSeparator = styled.div`
+	margin-left: 10px;
+	margin-right: 10px;
+	font-family: ${(props) => props.theme.fonts.regular};
+	color: ${(props) => props.theme.colors.selectedTheme.gray};
+`;
+
+const TraderText = styled.a`
+	font-family: ${(props) => props.theme.fonts.regular};
+	color: ${(props) => props.theme.colors.selectedTheme.button.text.primary};
+
+	&:hover {
+		text-decoration: underline;
+	}
+`;
+
+const TableHeader = styled.div`
```

Comment:
Wondering if this is a standard table header component we can extract to make re-usable?

Reply:
Good point. Just pushed this.